src/main/src/main/java/org/geoserver/config/impl/GeoServerInfoImpl.java

```diff
@@ -47,6 +47,8 @@
     protected Boolean xmlExternalEntitiesEnabled = Boolean.FALSE;
     
     protected String lockProviderName;
+
+    protected WebUIMode webUIMode = WebUIMode.DEFAULT;
```

Comment:
Btw, works fine only if the pull is made of a single commit, otherwise you have to run it after each commit.